app/config/collections.php

```diff
@@ -5833,14 +5833,14 @@
                 '$id' => ID::custom('_key_name'),
                 'type' => Database::INDEX_KEY,
                 'attributes' => ['name'],
-                'lengths' => [768],
+                'lengths' => [256],
                 'orders' => [Database::ORDER_ASC],
             ],
             [
                 '$id' => ID::custom('_key_signature'),
                 'type' => Database::INDEX_KEY,
                 'attributes' => ['signature'],
-                'lengths' => [768],
+                'lengths' => [256],
```

Comment:
How do we plan to migrate these indexes on cloud?

Reply:
Drop and recreate, and alter the length in metadata